src/NUnitFramework/tests/Assertions/WarningTests.cs

```diff
@@ -223,7 +223,12 @@ private static async Task<int> ThrowExceptionGenericTask()
         [TestCase(nameof(WarningFixture.WarningSynchronous), 1)]
         [TestCase(nameof(WarningFixture.WarningInThreadStart), 2)]
         [TestCase(nameof(WarningFixture.WarningInBeginInvoke), 5, ExcludePlatform = "mono", Reason = "Warning has no effect inside BeginInvoke on Mono")]
+#if NET7_0_OR_GREATER
+        [TestCase(nameof(WarningFixture.WarningInThreadPoolQueueUserWorkItem), 4, ExcludePlatform = "MacOSX")]
+        [TestCase(nameof(WarningFixture.WarningInThreadPoolQueueUserWorkItem), 5, IncludePlatform = "MacOSX")]
+#else
```

Comment:
Is this because of "InvokeStub_" lines not removed here?

Reply:
Not exactly, I described it here - https://github.com/nunit/nunit/issues/4170#issuecomment-1272403746